pkg/solana/types.go

```diff
@@ -61,16 +71,17 @@ type Config struct {
 	BillingAccessController   solana.PublicKey
 	MinAnswer                 bin.Int128
 	MaxAnswer                 bin.Int128
-	Decimals                  uint8
 	Description               [32]byte
+	Decimals                  uint8
 	F                         uint8
+	Round                     uint8
+	Padding0                  uint8
```

Comment:
```suggestion
	_ /* _padding0 */         uint8
```

Reply:
I'm not sure this will work since the `binary` encoding library can only see public fields

Reply:
[The docs](https://golang.google.cn/pkg/encoding/binary) say it should possibly be ok:

> When reading into structs, the field data for fields with blank (_) field names is skipped; i.e., blank field names may be used for padding. When reading into a struct, all non-blank fields must be exported or Read may panic.

> When writing structs, zero values are written for fields with blank (_) field names.

Reply:
We're using a binary library that is used in `solana-go` https://github.com/gagliardetto/binary
In this case, it will skip `_` parameters